### ISSUE TITLE
Planes from two points

### ISSUE THREAD

**Issue by winkerVSbecks** [mikolalysenko/split-polygon#2]
I understand that the polygon is represented by a set of co-ordinates however, I am not familiar with this syntax for planes. 

I am trying to split a 2d polygon with a line. I know that the line passes through (x1, y1) and (x2, y2). How would I use that information to get the `[u, v, w]` definition of the plane?

**Comment by blissvisitor** [comment]
I have the same question

**Comment by andybak** [comment]
Yeah. I'm here trying to port this to C# but I'm stuck on understanding how a 3 element vector represents a plane.

And considering this is about 2D polygons I'm a bit confused why the talk is about planes rather than line segments.

Going to dive into the code to try and figure it out but I wonder if there's some assumptions in the docs that aren't stated?

EDIT - Reading through #1 - that seems to be saying that a 3-vector for the plane represents (x, y, theta).

The readme states this code "Works in arbitrary dimensions" so I realise point above about 2D polygons isn't correct. I'm not clear how the (x, y, theta) notation generalises above 2 dimensions though.

EDIT2 - one obvious representation of a plane in 3 dimensions is (normal, distanceFromOrigin) - this would be a 4 vector in 3 dimensions and therefore a 3 vector in 2 dimensions. This makes more sense than the (x, y, theta) theory above.
